src/core/builder.ts

```diff
@@ -46,6 +46,10 @@ class DepGraphBuilder {
     this._pkgManager = pkgManager;
   }
 
+  public get packages(): { [pkgId: string]: types.PkgInfo } {
```

Comment:
something that might be confusing here, is that the `getPkgs()` on the `DepGraph` class returns a list, and here it's a map - maybe worth sticking to same convention just for consistency?

Reply:
@michael-go that's a very good point actually, I hadn't considered that. I'll update shortly.